data-miner/src/index.ts

```diff
@@ -178,10 +178,20 @@ app.get('/:sheetid/:gid', async (req: any, res: any) => {
 function makeASCTBData(data: any) {
   return new Promise((res, rej) => {
     let rows = [];
-    const headerRow = 11;
+    let headerRow = 0;
     const dataLength = data.length;
 
     try {
+      outerloop: for (let i = headerRow; i < dataLength; i++) {
+        innerloop: for (let j = 0; j < data[0].length; j++) {
```

Comment:
You don't need an inner loop like this. AS/1 _must_ be the first column.

Reply:
Cool. I will make the changes.

Reply:
Just so its clear what I'm thinking is roughly this:

```javascript
for (let i = headerRow; i < dataLength; i++) {
  if (data[i].length > 0 && data[i][0] === 'AS/1') {
    headerRow = i;
    break;
  }
}
```